src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -662,8 +900,7 @@ define(function (require) {
     TileMap.prototype.destroy = function () {
       if (this.popups) {
         this.popups.forEach(function (popup) {
-          popup.elem.off('mouseover').off('mouseout');
-          popup.layer.unbindPopup(popup.elem);
+          popup.off('mouseover').off('mouseout');
```

Comment:
From http://leafletjs.com/reference.html#events-removeeventlistener
 > If no function is specified, it will remove all the listeners of that particular event from the object"

This should be fine